sematic/resolvers/local_resolver.py

```diff
@@ -47,6 +49,8 @@ class LocalResolver(SilentResolver):
         entire pipeline again.
     """
 
+    _resource_manager: AbstractResourceManager = CloudResourceManager()
```

Comment:
Is this intentionally `CloudResourceManager` instead of `InMemoryResourceManager`?

Reply:
yes, we want the LocalResolver to store the info about the resources in the DB. That way the UI can display information about the resources.

Reply:
Ah I see. In that case, should it be named something else? E.g. `DBResourceManager` or `PersistedResourceManager` or other... ?

Reply:
Renamed `ServerResourceManager`

Reply:
renamed to ServerManager